### ISSUE TITLE
Issues building docker image

### ISSUE THREAD

**Issue by mflopez79** [chucknorris-io/chuck-api#17]
Hello,

I was trying to build the docker image using the docker-compose file provided by this project, and came across this error

```
ERROR: Service 'postgres' failed to build: The command '/bin/sh -c aws s3 cp s3://io.chucknorris.api/database/latest.sql /docker-entrypoint-initdb.d/latest.sql' returned a non-zero code: 255
```

In fact, trying to execute this command by hand
```bash
→ aws s3 cp s3://io.chucknorris.api/database/latest.sql .
fatal error: An error occurred (403) when calling the HeadObject operation: Forbidden
```

Would you please make `latest.sql` available again?

Thanks a lot

**Comment by matchilling** [comment]
Hi @mflopez79 👋, thanks for getting in touch and sorry for my delayed reply. Unfortunately I'm not planning to open source the database at the moment. But feel free to message me and tell me more about your specific your use case. Maybe there's a chance for collaborating on a new feature for the app.

Bests,
Mathias

**Comment by mflopez79** [comment]
This is not related to open source the database, it's a step on the docker building that fails (downloading `latest.sql`). Maybe that's not the database per se, only the schemas, I did a guess based on the filename.

tl;dr: docker build is broken and cannot be built

**Comment by matchilling** [comment]
Hey,

nope, the docker build fails as the resource the AWS S3 agent is trying to
retrieve is private. The db schema can be found here `
https://github.com/chucknorris-io/chuck-db` though. I'll remove the ref in
the Dockerfile and will just add some example.

Bests,
Mathias

**Comment by mflopez79** [comment]
Perfect, thanks!
